build.gradle

```diff
@@ -25,14 +24,19 @@ allprojects {
         repositories {
             google()
             jcenter()
+            mavenLocal()
             mavenCentral()
             maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
         }
+        dependencies {
+            classpath "com.badlogicgames.gdx:gdx-jnigen-gradle:2.0.0-SNAPSHOT"
```

Comment:
Lets do a release before we move to this?

Reply:
Probably a good idea, there's nothing I'm aware of that needs done before release. There's a few small tweaks/features which might be good, but they're not worth holding it back for.